### PR TITLE
ps1 mode: Logging and old code cleanup

### PR DESCRIPTION
**Pull request by kenshen112**
This is a quick PR to remove the incorrect and since unused reset function for ps1 mode that was commented out in CDVD as well as to clean up log statements in CDVD and cdrom

### CODE REVIEW

File: pcsx2/CDVD/CDVD.cpp [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #include "IopCommon.h"
 #include "AppConfig.h"
 
+#include <bitset>
```
Comment (refractionpcsx2) [inline pcsx2/CDVD/CDVD.cpp:20]: What was this added for?

Comment (kenshen112) [reply pcsx2/CDVD/CDVD.cpp:20]: Oops. When I was printing the binary values of the reg I used cout and bitset. I'll clean that up real quick

Comment (refractionpcsx2) [reply pcsx2/CDVD/CDVD.cpp:20]: plz learn to read hex ;)

Comment (kenshen112) [reply pcsx2/CDVD/CDVD.cpp]: It's been removed :p

**Pull request merged**